data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -824,23 +919,22 @@
 
 # From CDR V8, Curation receives HealthPro deceased records from RDR. We must ensure the incoming records follow the requirement.
 # Here is the highlight of the technical requirement of the incoming `death` records from RDR.
-# - Person_id and death_type_concept_id are populated
```

Comment:
Hi @brendagutman! From here below it looks like it's reverting the changes I made regarding `aou_death`. Your branch might contain some unrelated commits 🤔